src/Entities/Boat.h

```diff
@@ -30,30 +42,46 @@ class cBoat :
 	virtual void Tick(std::chrono::milliseconds a_Dt, cChunk & a_Chunk) override;
 	virtual void HandleSpeedFromAttachee(float a_Forward, float a_Sideways) override;
 
-	cBoat(double a_X, double a_Y, double a_Z);
+	cBoat(double a_X, double a_Y, double a_Z, eMaterial a_Material);
 
 	int GetLastDamage(void) const { return m_LastDamage; }
 	int GetForwardDirection(void) const { return m_ForwardDirection; }
 
 	float GetDamageTaken(void) const { return m_DamageTaken; }
 
-	int GetType(void) const { return m_Type; }
+	// tolua_begin
+
+	/** Returns the eMaterial of the boat */
+	eMaterial GetMaterial(void) const { return m_Material; }
+
+	/** Sets the eMaterial of the boat */
+	void SetMaterial(cBoat::eMaterial a_Material) { m_Material = a_Material; }
+
+	/** Returns the eMaterial for the passed item. Returns bmOak on failure */
+	static eMaterial GetMaterialFromItem(const cItem & a_Item);
```

Comment:
Most of our code uses `Type1ToType2` function names for such static conversions (`ItemTypeToString`, `StringToInteger` etc.), I think it makes more sense, the `Get` part is not needed.